firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLiteSchema.java

```diff
@@ -336,24 +336,32 @@ private void createV1RemoteDocumentCache() {
         });
   }
 
-  // TODO(indexing): Put the schema version in this method name.
-  private void createLocalDocumentsCollectionIndex() {
+  private void createFieldIndex() {
+    // Create two tables to support document indexing. `index_configuration` holds the
+    // configuration for all currently active indices. All entries apply for all users and it is
+    // not possible to only enable indices for a subset of users. `field_index` holds the index
+    // values. If there are pending mutations that affect an indexed field, an additional index
+    // entry is created per mutated field.
     ifTablesDontExist(
-        new String[] {"collection_index"},
+        new String[] {"index_configuration", "field_index"},
```

Comment:
Looks like other tables use plural.

Reply:
And what about `field_index_entries`, too long?

Reply:
I don't have a strong opinion. I made this "index_entries", which is shorter but still contains your suggestion. It also uses the same prefix as "index_configuration", which can be nice for grouping.